### PR TITLE
13 may assignment

### PR DESCRIPTION
**Pull request by sakshamtothenew**
things done in this assgnment:
1. testing added 
2.  tempory authh removed

### CODE REVIEW

File: expresssession/test/Student.test.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+let should = chai.should();
+
+
+chai.use(chaiHttp);
+
+
+describe('Students' , () => {
+
+    beforeEach((done) => {
+        Student.remove({} , (err)=> {
```
Comment (Pooja-22) [inline expresssession/test/Student.test.js:17]: Why are we doing this?

File: expresssession/test/Student.test.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+    describe('save students' , () => {
+        it("should save students in database" , (done) => {
+            let student = {
+                firstname : "sakshaam" , 
+                lastname : "sachdeva" , 
+                branch : "CSE"
+            }
+            chai.request(server)
+            .post('/student')
```
Comment (Pooja-22) [inline expresssession/test/Student.test.js:42]: Is this hitting actually API

Comment (sakshamtothenew) [reply expresssession/test/Student.test.js:42]: yes

File: expresssession/test/Student.test.js (hunk 1 above)
Comment (sakshamtothenew) [reply expresssession/test/Student.test.js:17]: in order to remove everything from database stuudent however , here we were supposse to use testing database but this demo app so i used the same

**Pull request merged**